nostalgia/sources/spotify.py

```diff
@@ -0,0 +1,29 @@
+import just
+import pandas as pd
+
+from nostalgia.ndf import NDF
+from nostalgia.times import datetime_from_format
+
+
+class Spotify(NDF):
+    @classmethod
+    def load(cls, file_path="/Users/nikolay_vasilishin/nostalgia_data/input/spotify/StreamingHistory0.json", nrows=None):
```

Comment:
I'm sure we want to make this path more general, most likely fitting with the `~/nostalgia_data/input/<service-name>/...`

Also, (and I'm not sure), do we only want to import `StreamingHistory0.json`?  Or `StreamingHistory*.json`?

Reply:
Indeed, good point. You can see in the documentation we have that there is a helper for this:

https://nostalgia-dev.github.io/nostalgia/docs/adding_new_data/add_or_contribute_a_new_data_source/#loading-data

specifically: `load_dataframe_per_json_file`

See it in the Samsung source:

https://github.com/nostalgia-dev/nostalgia/blob/master/nostalgia/sources/samsung/heartrate.py#L29

Reply:
@jjc385 And thanks here. About paths - yes, it's still in wip, just wanted to align with @kootenpv and you (as you joined :))
For `StreamingHistory0.json` it's a good point, not sure if they reflected possible names for this file, but seems logical if they split large history into several ones.